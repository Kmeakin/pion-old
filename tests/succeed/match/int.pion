//~ run = "${PION} elab ${FILE}"

fun (x : Int) => match x {
    0 => true,
    _ => false,
}
