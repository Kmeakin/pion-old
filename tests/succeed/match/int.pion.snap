exit status: 0
stdout:
fun (x : Int) => match x {0 => true, _ => false} : Int -> Bool

stderr:
